automation/check-patch.setup.sh

```diff
@@ -18,6 +18,6 @@ gimme_dir=$tmp_dir/go/gimme
 mkdir -p $gimme_dir
 curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=${gimme_dir} bash >> ${gimme_dir}/gimme.sh
 source $gimme_dir/gimme.sh
-
+export E2E_TEST_ARGS='-ginkgo.noColor'
```

Comment:
Please keep this in the original place. We should keep our crazy Makefile parameters as close to the Makefile as possible, not to spread them everywhere.